docs/deploy/confluent-cloud.md

```diff
@@ -143,6 +143,39 @@ springKafkaConfigurationOverrides:
 
 Then simply apply the updated `values.yaml` to your K8s cluster via `kubectl apply`. 
 
+#### DataHub Actions
+
+Configuring Confluent Cloud for DataHub Actions requires some additional edits to your `executor.yaml`. Under the Kafka
+source connection config you will need to add the Python style client connection information:
+
+```yaml
+    connection:
+        bootstrap: ${KAFKA_BOOTSTRAP_SERVER:-localhost:9092}
+        schema_registry_url: ${SCHEMA_REGISTRY_URL:-http://localhost:8081}
+        consumer_config:
+          security.protocol: ${KAFKA_PROPERTIES_SECURITY_PROTOCOL:-PLAINTEXT}
+          sasl.mechanism: ${KAFKA_PROPERTIES_SASL_MECHANISM:-PLAIN}
+          sasl.username: ${KAFKA_PROPERTIES_SASL_USERNAME}
+          sasl.password: ${KAFKA_PROPERTIES_SASL_PASSWORD}
+```
+
+Specifically `sasl.username` and `sasl.password` are the differences from the base `executor.yaml` example file.
+
+Additionally, you will need to set up secrets for `KAFKA_PROPERTIES_SASL_USERNAME` and `KAFKA_PROPERTIES_SASL_PASSWORD`
```

Comment:
So is the below relevant on helm? If yes lets add that it's only relevant for helm (not docker compose)

Reply:
The settings are relevant for both, but yes this specific way of setting things up is for helm. For docker-compose they'll have to set up the env vars, I can add that.